### PR TITLE
Fix the aggregate losses bug

### PR DESCRIPTION
**Pull request by micheles**
See Anirudh's comment to https://github.com/gem/oq-engine/pull/1863. The tests are green: https://ci.openquake.org/job/zdevel_oq-risklib/1125/

### CODE REVIEW

File: openquake/calculators/scenario_risk.py [hunk 1]
```diff
@@ -73,11 +74,10 @@ def scenario_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
...
             stats[:, r, 3] = out.insured_loss_matrix.std(ddof=1, axis=1)
-            avg = result[lti]
             for asset, stat in zip(out.assets, stats):
-                avg['avg', r, asset.idx] = stat
-            result[lti]['agg', r, 0] = out.aggregate_losses
-            result[lti]['agg', r, 1] = out.insured_losses
+                result['avg'].append((l, r, asset.idx, stat))
+            result['agg'][:, l, r, 0] += out.aggregate_losses
+            result['agg'][:, l, r, 1] += out.insured_losses
     return result
```
Comment (raoanirudh) [inline openquake/calculators/scenario_risk.py]: Please check if the nature of the return value mentioned in the docstring matches the actual value returned by this function.

Comment (micheles) [inline openquake/calculators/scenario_risk.py]: Updated the docstring

File: openquake/calculators/scenario_risk.py [hunk 2]
```diff
@@ -47,20 +47,26 @@ def scenario_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
     :param monitor:
         :class:`openquake.baselib.performance.PerformanceMonitor` instance
     :returns:
-        a dictionary (key_type, loss_type) -> losses where the `key_type` can
-        be "agg" (for the aggregate losses) or "ins" (for the insured losses).
+        a dictionary {
+        'agg': array of shape (E, R, 2),
```
Comment (raoanirudh) [inline openquake/calculators/scenario_risk.py]: I think the number of loss types is missing in the shape?

**Comment by raoanirudh** [comment]
LGTM

File: openquake/calculators/scenario_risk.py (hunk 2 above)
Comment (micheles) [inline openquake/calculators/scenario_risk.py]: You are too careful! I updated the docstring with an obsolete version of the code :-(
Now it is fine.

**Pull request merged**